### PR TITLE
Add CONTRIBUTING.md and SECURITY.md

### PR DESCRIPTION
**Pull request by Neriderc**
Added the CONTRIBUTING.md and SECURITY.md files and updated README.md with reference to CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Please note this project is [AGPLv3 licenced](https://www.gnu.org/licenses/agpl.html), and all contributions are under the same licence. Please ensure all work is your own.
+
+If you're new to contributing, see [this tutorial](https://github.com/firstcontributions/first-contributions) on contributing to projects on GitHub.
+
+Check out the [Issues](https://github.com/zusam/zusam/issues) for things that need attention.
+
+### Pull requests
+
+Please target any pull requests to the `next` branch. This branch will be copied to `master` with each release.
```
Comment (nrobinaubertin) [inline CONTRIBUTING.md:34]: "copied" should be "merged" instead

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Next, compile the webapp and copy to the public directory with:
+`npm run build; rm -r ../public/*.{js,css,map,png}; cp -r dist/* ../public/`
+
+Now install the composer dependencies by running:
+`cd /zusam/api && php bin/composer install`
+
+Now exit the container with ctrl + D and then run the container using repo files using:
+`dev/start-test-container`
+
+You should now be able to access Zusam on http://localhost:8080
```
Comment (nrobinaubertin) [inline CONTRIBUTING.md:61]: You could create a markdown link from this

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Any code updates you make should be reflected immediately in this running instance.
```
Comment (nrobinaubertin) [inline CONTRIBUTING.md:63]: This is true when doing backend work but not for the frontend when the files need to be build and moved again.  
But yes, any change in the repository, done on the host or from the dev container will land in the test container.

**Comment by nrobinaubertin** [comment]
It can be worth adding that the database created by default will provide a "zusam" user with the password "zusam"

File: CONTRIBUTING.md (hunk 3 above)
Comment (Neriderc) [reply CONTRIBUTING.md:63]: So after making changes in the /app directory, you should run this command again?
`npm run build; rm -r ../public/*.{js,css,map,png}; cp -r dist/* ../public/`

Comment (nrobinaubertin) [reply CONTRIBUTING.md:63]: Yes

**Comment by Neriderc** [comment]
I've made the requested changes in a new commit added to this PR. Please double check to make sure I've understood correctly.

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**